clients/src/main/java/org/apache/kafka/common/errors/RebalanceInProgressException.java

```diff
@@ -20,7 +20,9 @@
     private static final long serialVersionUID = 1L;
 
     public RebalanceInProgressException() {
-        super();
+        super("Request cannot be completed since the consumer group is rebalancing. This can happen when a group has started, " +
```

Comment:
Why not have this extend `RetriableCommitFailedException`?

Reply:
Good call. Actually I think we should just use this one for retriable commit failure since today the `RebalanceInProgressException` is used in other responses, like join/sync and heartbeat. I would refactor my code to use `RetriableCommitFailedException` directly.

Reply:
Sounds good